src/DataStructures/Python/Bindings.cpp

```diff
@@ -8,9 +8,11 @@ namespace py = pybind11;
 namespace py_bindings {
 void bind_datavector(py::module& m);  // NOLINT
 void bind_matrix(py::module& m);      // NOLINT
+void bind_tensordata(py::module& m);  // NOLINT
```

Comment:
`TensorData.hpp` is in the `Tensor` subdirectory, so please move these bindings to `DataStructures/Tensor/Python` (which doesn't exist yet). You can copy over the `CMakeLists.txt` and `Bindings.cpp` from `DataStructures/Python` and adjust them slightly.

Reply:
Discussed with @isha1810 and we were a bit confused by how this would work in practice .  Despite the fact that `TensorData.hpp` is in a different subdirectory, it is still included into the `DataStructures` library, so it seems like the tensor data python bindings should probably be a part of the `DataStructures` python module.  We weren't sure if there exists a function to extend a python module (analogous to `target_spectre_sources`) so that the module could be created in one place and extended elsewhere.  A possible alternative might be to have the file thats currently at `DataStructures/Python/TensorData.cpp` moved to `DataStructures/Tensor/Python/TensorData.cpp` but then still use the same `Bindings.cpp` file to bind it.  Let us know what you think!

Reply:
Ah I didn't realize that the two classes in question are in the `Tensor` subdirectory but of course we don't have a separate `Tensor` library. I agree with you that the bindings should remain in the `DataStructures` Python module, and I think that indeed the only file that should move is `DataStructures/Python/TensorData.cpp` to `DataStructures/Tensor/Python/TensorData.cpp`, as you suggested.

Reply:
It will go in `DataStructures/Python/Tensor/TensorData.cpp`

Reply:
sounds good as well